### PR TITLE
SolarWinds Service Desc Connector - change to update_at

### PR DESCRIPTION
**Pull request by LiranRomSamanage**
We would like to notify our customer about a change we did in the connector 
Refer moving the filter to `update_at`

### CODE REVIEW

**Review by giladsmg — APPROVED** [review]

**Comment by LiranRomSamanage** [comment]
@DougKlopfenstein please approve

**Review by DougKlopfenstein — APPROVED** [review]

**Comment by Court72** [comment]
@DougKlopfenstein

Thanks for reviewing the PR. When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by DougKlopfenstein** [comment]
#sign-off

File: powerquery-docs/connectors/solarwinds-service-desk.md [hunk 1]
```diff
@@ -32,13 +32,22 @@ Before you can use this connector to get SolarWinds Service Desk data, you must
...
+
 The following instructions contain special steps for customers who previously used the BETA SWSD Power BI connector to create reports and want to connect those reports with the new out-of-the-box version.
 
 1. In Power BI Desktop, select **Get Data** from **Home**. Select **SolarWinds Service Desk**, and then select **Connect**.
 
    :::image type="content" source="./media/solarwinds-service-desk/get-data.png" alt-text="Screenshot of Get Data from SolarWinds Service Desk.":::
 
 2. Sign in with the JSON web token you generated as described in prerequisites, and then select **Connect** to verify your access to SWSD.
+> [!IMPORTANT]
+> If you have an ESM account, the JSON token you use for the connector must be generated by a Service Provider Administrator.
```
Comment (Jak-MS) [inline powerquery-docs/connectors/solarwinds-service-desk.md:50]:
```suggestion
   > [!IMPORTANT]
   > If you have an ESM account, the JSON token you use for the connector must be generated by a Service Provider Administrator.
```

**Comment by Jak-MS** [comment]
@LiranRomSamanage - can you commit my suggestion and tag me again ... otherwise this is ready to merge.

File: powerquery-docs/connectors/solarwinds-service-desk.md (hunk 1 above)
Comment (LiranRomSamanage) [reply powerquery-docs/connectors/solarwinds-service-desk.md]: done @Jak-MS

**Comment by LiranRomSamanage** [comment]
@Jak-MS please approve

**Comment by DougKlopfenstein** [comment]
Closing and reopening to rebuild.

**Pull request merged**